apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -128,6 +140,25 @@ class EditSectionForm extends Component {
     );
   };
 
+  // valid event names: 'Section Setup Complete', 'Section Setup Cancelled'.
+  recordSectionSetupExitEvent = eventName => {
+    const {section, courseOfferings, isNewSection} = this.props;
+
+    const courseName = courseOfferings.hasOwnProperty(section.courseOfferingId)
+      ? courseOfferings[section.courseOfferingId].display_name
```

Comment:
Just checking that we want the display_name here (which will change based on locale) and not the id?

(I added this comment below then realized it belonged here -- sorry for the duplicate)

Reply:
Updated based on slack discussion in e05f83c.